recipes/mold/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.45.0"
+
+class MoldConan(ConanFile):
+    name = "mold"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rui314/mold/"
+    license = "Zlib"
+    description = ("mold is a faster drop-in replacement for existing Unix linkers. It is several times faster than the LLVM lld linker")
+    topics = ("mold", "ld", "linkage", "compilation")
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    generators = "make"
+
+    def validate(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration('{} can\'t be built on {}.'.format(self.name, self.settings.os))
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) <= "10":
+            raise ConanInvalidConfiguration("GCC version 10 or higher required")
+        if (self.settings.compiler == "clang" or self.settings.compiler == "apple-clang") and tools.Version(self.settings.compiler.version) <= "11":
+            raise ConanInvalidConfiguration("Clang version 11 or higher required")
+        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch :
+            raise ConanInvalidConfiguration('Use apple-clang does not work on ARM with this recipe')
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _patch_sources(self):
+        zlib_include_path = self.deps_cpp_info["zlib"].rootpath
+        zlib_include_path = os.path.join(zlib_include_path, "include")
+
+        openssl_include_path = self.deps_cpp_info["openssl"].rootpath
+        openssl_include_path = os.path.join(openssl_include_path, "include")
+
+        xxhash_include_path = self.deps_cpp_info["xxhash"].rootpath
+        xxhash_include_path = os.path.join(xxhash_include_path, "include")
+
+        tools.replace_in_file("source_subfolder/Makefile", "-Ithird-party/xxhash ", "-I{} -I{} -I{}".format(xxhash_include_path, zlib_include_path, openssl_include_path))
+
+    def requirements(self):
+        self.requires("zlib/1.2.12")
+        self.requires("openssl/1.1.1q")
+        self.requires("xxhash/0.8.1")
```

Comment:
`onetbb` and `mimalloc` are also on CCI. Please patch the makefile and use those instead of the vendored sources.

Reply:
done

Reply:
You'll have to patch this line in the Makefile as well

https://github.com/rui314/mold/blob/7f7d10cc896e0c7513d09723d5ca3d8346bdea49/Makefile#L127

since this will build onetbb and mimalloc.

Reply:
done, please resolve